torch/_prims/utils.py

```diff
@@ -671,7 +671,7 @@ def check_same_dtype(*args):
 }
 
 
-def _get_computation_dtype(dtype: torch.dtype) -> torch.dtype:
+def get_computation_dtype(dtype: torch.dtype) -> torch.dtype:
```

Comment:
Sure -- can we use the wrappers and utils in wrappers.py more?

Reply:
In this case it's tricky to use the `elementwise_type_promotion` wrapper - is there another one you're thinking of here? Are you thinking of `_maybe_convert_to_dtype`?

Reply:
Seems like that should be a drop-in replacement for `maybe_cast`?

Reply:
Doesn't work for `None` - I can add that case and then unify?